.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     - uses: actions/checkout@master
     - name: Regula
       id: regula
-      uses: fugue/regula-action@v0.1.0
+      uses: fugue/regula-action@report-message
```

Comment:
Do you want to stick to versioned releases of regula-action?

Reply:
Yes, this is just so I can try it out in this repo, I'll use a versioned release once the corresponding code in `regula-action` gets merged.